### DISCUSSION TITLE
Can't add more disk 1.2.8

### DISCUSSION THREAD

**Discussion by OMeliante** [Chia-Network/chia-blockchain#8674] [Support]
First of all, the 1.2.8 update shows as 1.2.6-dev67
![Screenshot_2021-10-01-13-16-11-710_com anydesk anydeskandroid](https://user-images.githubusercontent.com/68663977/135664314-1c1dd0af-cf6a-4116-a7ab-040379a5ffba.jpg)

And can not insert more disks. I am trying to insert path C:\DISCOS\8\novos, at GUI and even editting config.yaml and doesn't work
![Screenshot_2021-10-01-14-31-17-499_com anydesk anydeskandroid](https://user-images.githubusercontent.com/68663977/135664564-daad034a-dde5-4d80-9cf9-c140e72156ae.jpg)
![Screenshot_2021-10-01-14-31-42-748_com anydesk anydeskandroid](https://user-images.githubusercontent.com/68663977/135664570-80e5237d-9f7a-4c2a-97dd-0ca9cbcb6ac6.jpg)

**Comment by loppefaaret** [comment]
that list has a scroll bar on the right...

**Reply by OMeliante** [reply]
That's basics. I know that of course.
Devs launched 1.2.9 so its solved
